client/src/Components/FloatingButton/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import './index.css'
+
+//still need to add button code so that it spins onClick not
+//just on hold
+
+export const FloatingButton = () => {
+  return (
+    <nav>
+      <ul id='list'>
+        <li>Home</li>
+        <li>About Us</li>
+        <li>Contact Us</li>
+        <li>Resources</li>
+      </ul>
+    </nav>
+  )
+}
+
+export const Navbar: React.FC = () => {
+  const [open, setOpen] = useState(false)
+
+  const handleClick = () => {
+    setOpen(!open)
+    const navbar = document.getElementById('navbar')
+    if (open) {
+      navbar?.classList.remove('navbarOpen')
+      navbar?.classList.add('navbarClose')
+    }
+    if (!open) {
+      navbar?.classList.add('navbarOpen')
+      navbar?.classList.remove('navbarClose')
+    }
+  }
```

Comment:
Why is vanilla js used here when the state change can be done directly using react.